docs/vars.md

```diff
@@ -170,6 +170,11 @@ The possible vars are:
 * *kubelet_custom_flags*
 * *kubelet_node_custom_flags*
 
+As an alternative, these very same parameters can be set as key-value pairs of configuration parameters that will be inserted into the kubelet YAML config file:
```

Comment:
It must be noted that the use of flags is deprecated
and you must use config args

Reply:
@LuckySB Are you suggesting that the doc should reflect the fact that flags are deprecated and the extra_args should be used instead ?

Reply:
Yes. I am sure about that.
Many flags are deprecated and we can tell the user this fact.
https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/

This is not an alternative; this is the new correct way to configure